### DISCUSSION TITLE
Cannot get data from a row in the Table component to put in Row Action

### DISCUSSION THREAD

**Discussion by hkim-jadehealth** [Budibase/budibase#15485] [Help]
We need to implement the below steps:

Load data from REST API to Table component.
Each row in the table needs to have a Delete and Edit button:
- Edit button: On-click, needs to show the Modal with info from the row that was clicked on
- Delete button: On-click, this should execute an API to delete that row by id of that record

My problems are:
- For Edit button, I binded from SelectedRow in Table to form fields in Modal; while designing, the data shows correctly, but in Preview, this doesn't work.
- For Delete button, similar to the Edit button where that I binded id field of SelectedRow in the Table to my API request, but it didn't work.

It seems the data of SelectedRow couldn't show up in Modal or send to Action of Delete button.

**Comment by andz-bb** [comment]
hey @hkim-jadehealth 

for the table button actions, you should use the `{{ Clicked row }}` binding rather than the selected row binding.

If you are still having issues with this, please send us an app export and we can investigate further.

**Comment by ledzepff** [comment]
I'm experiencing the same issue with modal form. Here is the stack of components I have

![image](https://github.com/user-attachments/assets/c6512a60-75f9-47b7-a62b-46ef2453058e)

The regions_field_form is connected to regions data-source (table in mysql)

![image](https://github.com/user-attachments/assets/7eadc5c1-0e5e-4d60-9b18-1391cafd234a)

Here is the model form

![image](https://github.com/user-attachments/assets/06ab4fcf-3abc-4e98-ad79-35af18231d49)

And two actions on selected row in super table

![image](https://github.com/user-attachments/assets/1492725b-f4e7-4ffd-8e82-adbe2be4b8d2)

The "name" TextField is set to name 

![image](https://github.com/user-attachments/assets/f8aeb7ce-0727-4522-bc5d-362958c0ee47)

However, after modal is open, the name field doesn't get value. The most interesting part if that if I do the same without modal window, just take form our of the modal, it's working. It also doesn't work with side panel. 


https://github.com/user-attachments/assets/d3525573-8a03-49a4-b15c-480f59c43e80

**Comment by ledzepff** [comment]
Here is the other video, which shows that when another form is our of modal window, the field value is successfully set


https://github.com/user-attachments/assets/c63a9144-6adf-4b34-918a-b86e2a492795

This is very serios bug since it doesn't allow to deisgn your app is a way you need!
